.changeset/clean-moose-retire.md

```diff
@@ -0,0 +1,6 @@
+---
+'@finos/legend-application-studio-deployment': patch
+'@finos/legend-application-studio-bootstrap': patch
+---
+
+Support for Mastery Precedence rules
```

Comment:
You don't need changeset message for these modules.
I think you missed the only relevant changeset which is "@finos/legend-extension-dsl-mastery"

Reply:
I had used the command 'yarn changeset' to generate the changeset. Do you still want me to change it?

Reply:
yeah please change it

Reply:
Thanks. Both comments have now been addresseed.